cmd/tetragon/main.go

```diff
@@ -401,11 +406,18 @@ func tetragonExecute() error {
 
 	hookRunner := rthooks.GlobalRunner().WithWatcher(k8sWatcher)
 
+	redactionFilters, err := getRedactionFilters()
+	if err != nil {
+		return err
+	}
+	log.WithFields(logrus.Fields{"redactionFilters": redactionFilters}).Debug("Configured redaction filters")
```

Comment:
IMO, I think `Info` is better  here.

Reply:
Sure why not. For the record we're also dumping other filter types in debug logs. Maybe worth changing those too

Reply:
Updated